src/OmniSharp/Models/v1/DnxProject.cs

```diff
@@ -12,6 +13,7 @@ public class DnxProject
         public IList<string> Configurations { get; set; }
         public IList<string> ProjectSearchPaths { get; set; }
         public IList<string> Frameworks { get; set; }
+        public IList<DnxFramework> DnxFrameworks { get; set; }
```

Comment:
Agreed, but I didn't want to break anyone intentionally.  I'm not sure if it's a highly used property yet for editors though.